test/sql/visualizer/visualizer_tpch_sf001.test_slow

```diff
@@ -0,0 +1,46 @@
+# name: test/sql/visualizer/visualizer_tpch_sf001.test_slow
+# description: Test Visualizer for TPC-H SF0.1
+# group: [visualizer]
+
+
+require visualizer
+
+require tpch
+
+statement ok
+PRAGMA enable_profiling
+
+statement ok
+PRAGMA profiling_output='__TEST_DIR__/test.json'
+
+statement ok
+PRAGMA profiling_mode = detailed
+
+statement ok
+CALL dbgen(sf=0.1);
+
+loop i 1 10
+
+query I
+PRAGMA tpch(${i})
+----
+<FILE>:extension/tpch/dbgen/answers/sf0.1/q0${i}.csv
+
+statement ok
+Pragma visualize_last_profiling_output ='__TEST_DIR__/test.html'
```

Comment:
What if there is no last profiling output (i.e. this is run as the first query)?

Reply:
In those cases, the injected JSON file would be like  ```"{ \"result\": \"error\" }\n"```. It does not crash but also there is nothing to visualize.

Reply:
Could you verify that it doesn't crash through a test?